CHANGELOG.asciidoc

```diff
@@ -3,6 +3,21 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-7.13.1]]
+=== Beats version 7.13.1
+https://github.com/elastic/beats/compare/v7.13.0...v7.13.1[View commits]
+
+==== Bugfixes
+
+*Auditbeat*
+
+- Mitigate deadlock is aws-s3 input when SQS visibility timeout is exceeded. {issue}25750[25750]
+- Fix httpjson cursor override with empty values by adding `ignore_empty_value` option. {pull}25802[25802]
+
+*Filebeat*
+
+- Improve inode reuse handling by removing state for removed files more eagerly from the internal state table in the logs inputs. {pull}25756[25756]
```

Comment:
Interesting. The backport to 7.x is still pending on CI, so we will reintroduce that to `Changelog.next.asciidoc` when the backport gets merged. The scripts are able to remove the entries again, but this will require us to cleanup `Changelog.next.asciidoc` more regularly in master and 7.x. A multiple file based approach will suffer from the same problem. This reminds me, why we discussed to generate the changelog from the original PRs description.